files/index.js

```diff
@@ -1,3 +1,4 @@
+/* global document */
```

Comment:
Do we also need the copyright here?

Reply:
I don't think so. These files will be copied to the bootstrapped app, and I don't want to make any assumptions about the license of the new app.